tools/mapping/gen_mapping_files/runoff_to_ocn/README

```diff
@@ -3,21 +3,38 @@ ABOUT THIS TOOL
 ===============
 
 The src/ directory here contains F90 files and a Makefile to produce the
-runoff_map executable, which reads in a runoff file and an ocean grid file
-and outputs a map from the former to the latter:
-
-1) Computes an initial nearest neighbor mapping from the rof grid cells to
-   active ocean grid cells.
-2) Computes a smoothing matrix on the ocean grid.
-3) The two matrices are multipled together to generate the final mapping file.
+runoff_map executable. The executable reads runoff_map.nml and produces three
+maps:
+
+1) A nearest neighbor mapping from the rof grid to the ocean grid.
+   * The optional file_ocn_coast_mask file allows mapping to a subset of ocean
+     cells, presumably the coastal cells)
+2) A smoothing map from the ocean grid to the ocean grid.
+   * The optional file_ocn_coast_mask file allows mapping from a subset of
```

Comment:
I appreciate the detailed descriptions in the README file: thank you!

However, with these new options, I'm now confused about how I should actually create the mapping files needed by CESM. I'm especially unsure about when I should specify a file_ocn_coast_mask file.

Can we describe how to generate specific mapping files, either here or elsewhere? I'm especially interested in how to regenerate the GLC2OCN runoff mapping files (and maybe it would be best if we sat down and chatted about this to make sure we get it right), but presumably people would also like to know exactly how to generate rof2ocn_liq and rof2ocn_ice.